Core/Transaction.cs

```diff
@@ -1,47 +1,49 @@
-using Newtonsoft.Json;
 using System;
+using Newtonsoft.Json;
 
 namespace Core
 {
-    public class Transaction : IComparable<Transaction>
-    {
-        public string CardNumber { get; }
-        public string ApplicationCode { get; }
-        public DateTimeOffset TimeStamp { get; }
-        public Money Amount { get; }
-        public bool IsIncome { get; }
-        public bool IsExpence { get; }
-        public Money Rest { get; }
-        public string Terminal { get; }
-        public string Description { get; }
+	public class Transaction : IComparable<Transaction>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)